master/lib/Munin/Master/Node.pm

```diff
@@ -151,13 +151,19 @@ sub _do_connect {
     return 1;
 }
 
+
+sub _get_node_or_global_setting {
+    my ($self, $key) = @_;
+    return exists $self->{configref}->{$key} ? $self->{configref}->{$key} : $config->{$key};
```

Comment:
using `defined()` is usually better than `exists`.

But, I agree... here the original line was using `exists`, so it might be safer for _this_ particular change.